packages/frontend-shared/src/warning/Warning.vue

```diff
@@ -64,6 +66,11 @@ let message = computed(() => {
     return [props.message, `        ${ props.details }`].join('\n\n')
   }
 
+  if (props.helpLinkHref) {
+    // eslint-disable-next-line prefer-template
+    return props.message + ` <span class="ml-[4px]"><a href="${props.helpLinkHref}" class="text-indigo-500 hocus-link-default">${t('links.learnMoreButton')}</a></span>`
```

Comment:
Is this just trying to shove a link into the message so it gets processed as markdown or whatever for the `v-html`? Why not just add a conditional `ExternalLink` in the template?

```vue
<div
  ref="markdownTarget"
  class="warning-markdown"
  v-html="markdown"
/>
<ExternalLink v-if="helpLinkHref" ..... >
```

Reply:
This needs to be added to the `props.message` so that the link display `inline` the `p` copy.

Reply:
Are we able to tack this on using markdown format rather than inline HTML? Would this get processed correctly by `useMarkdown`?

```
const learnMoreLabel = t('links.learnMoreButton')
return `${props.message} [${learnMoreLabel}](${props.helpLinkHref})`
```

Reply:
Ahhh, yes. Now i'm following you. Yes it is implemented in 27100e3c7a9d2ac96b08fb689bbb2b8b11877361